ui/src/app/icons/card-menu-icon/component.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import {mount} from 'enzyme';
+import * as React from 'react';
+
+import {
+  CardMenuIconComponentReact
+} from './component';
+
+// Note that the description is slightly different from the component for easier test filtering.
+describe('CardMenuIconComponent', () => {
```

Comment:
This could be a point for discussion, but I'm not sure this test is worth it - assuming this icon gets rendered somewhere in the application which has a test.

Reply:
(even if it isn't... meh). Honestly these icons could probably all be thrown into one (or a small handful of) file(s) once we do away with the Angular wrapper; the boilerplate involved here always bothered me.

Reply:
Once we have some control over webpack, I'd love to do what Terra does, and be able to import svg's directly into code.